src/data/dialogmote/dialogmote_actions.ts

```diff
@@ -1,65 +1,99 @@
-import { DialogmoteInnkallingDTO } from "./dialogmoteTypes";
+import { DialogmoteDTO, DialogmoteInnkallingDTO } from "./dialogmoteTypes";
 
-export const OPPRETT_INNKALLING_FORESPURT = "OPPRETT_INNKALLING_FORESPURT";
-export const OPPRETTER_INNKALLING = "OPPRETTER_INNKALLING";
-export const INNKALLING_OPPRETTET = "INNKALLING_OPPRETTET";
-export const OPPRETT_INNKALLING_FEILET = "OPPRETT_INNKALLING_FEILET";
-export const OPPRETT_INNKALLING_FULLFORT = "OPPRETT_INNKALLING_FULLFORT";
+export enum DialogmoteActionTypes {
+  OPPRETT_INNKALLING_FORESPURT = "OPPRETT_INNKALLING_FORESPURT",
+  OPPRETTER_INNKALLING = "OPPRETTER_INNKALLING",
+  INNKALLING_OPPRETTET = "INNKALLING_OPPRETTET",
+  OPPRETT_INNKALLING_FEILET = "OPPRETT_INNKALLING_FEILET",
+  OPPRETT_INNKALLING_FULLFORT = "OPPRETT_INNKALLING_FULLFORT",
+  FETCH_DIALOGMOTE = "FETCH_DIALOGMOTE",
```

Comment:
Pirk: skulle vi brukt HENT_* i dialogmote-redux-koden siden det er brukt for henting av data i andre reducers?

Reply:
Lurte litt på den der, ref diskusjonen vi hadde om språk. Skjønte det som at vi landet på at alt er på engelsk utenom domeneuttrykk/faguttrykk. Men vi kan jo ta det i plenum? For min del tenker jeg kanskje vi heller refaktorerer det gamle til engelsk

Reply:
Ja, san, så lenge vi er konsistente. Vi får bli enige om noe felles og refaktorere 👍